go.mod

```diff
@@ -3,8 +3,9 @@ module github.com/mattermost/mattermost-plugin-todo
 go 1.13
 
 require (
+	github.com/gorilla/mux v1.7.3
 	github.com/mattermost/mattermost-server/v5 v5.20.0
 	github.com/mholt/archiver/v3 v3.3.0
 	github.com/pkg/errors v0.9.1
-	github.com/stretchr/testify v1.5.1
+	github.com/stretchr/testify v1.5.1 // indirect
```

Comment:
?? // indirect

Reply:
I removed `plugin_test.go` that did nothing